### PR TITLE
rholangtut-0.3 (Spanish Translation) - Issue #651

### PR DESCRIPTION
**Pull request by ICA3DaR5**
## Overview
This is a professional translation of the rholangtut-0.3.md file in Spanish, which is one of the most spoken languages in the world.

### Does this PR relate to an RChain JIRA issue? 
If applicable, add link to corresponding JIRA issue.

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

File: docs/rholang/rholangtut-0.3_ES.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+    31           }
+    32         }
+    33       }
+    34     }
+    35   }
+    36 }
+
+1) Creamos un nuevo canal MakeCell y después lo usamos en la línea 3 como el nombre de un contrato interno. Ningún proceso que no sea el código dentro de este ámbito léxico puede invocarlo.
+
+3) El contrato `MakeCell` toma tres argumentos. El primer argumento es el valor inicial que seráa guardado en la celda. El segundo y tercer argumentos son canales en los cuales la celda recibirá solicitudes para obtener y establecer el valor. Tenga en cuenta que queremos que el primer argumento sea un proceso y el segundo y el tercero sean nombres, pero los nombres siempre se reciben por canales, por lo que debemos hacer que el primer argumento sea un patrón que comience con `@` para indicar que el nombre que recibimos como el primer argumento es un proceso citado y es ese el proceso que queremos vincular a la variable.
```
Comment (kjmcqueen) [inline docs/rholang/rholangtut-0.3_ES.md]: "...será guardado..."

File: docs/rholang/rholangtut-0.3_ES.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+
+3) La construcción `match` permite desestructurar una variable mediante la coincidencia de patrones.
+
+4) Los patrones de la lista facilitan la coincidencia con el resto de una lista. Si `list` coincide con el patrón de un par head/tail ejecutamos el cuerpo principal del ciclo.
+
+5) Creamos un canal para que el handler de elementos nos notifique que ya está hecho con el elemento actual.
+
+6) Llamamos al procesador en el elemento y el canal de acuse de recibo.
+
+7) Cuando recibimos el acuse de recibo, volvemos a llamar al iterador a la cola.
```
Comment (kjmcqueen) [inline docs/rholang/rholangtut-0.3_ES.md]: You sometimes use "head/tail" and sometimes "cola" etc.  Please be consistent in translated vs. literal terms.

File: docs/rholang/rholangtut-0.3_ES.md (hunk 1 above)
Comment (ICA3DaR5) [reply docs/rholang/rholangtut-0.3_ES.md]: Fixed.

File: docs/rholang/rholangtut-0.3_ES.md (hunk 2 above)
Comment (ICA3DaR5) [reply docs/rholang/rholangtut-0.3_ES.md]: Fixed as well!

**Comment by ICA3DaR5** [comment]
@kjmcqueen Could you please merge this, if there are no other issues with it? Thanks!

**Comment by KellyatPyrofex** [comment]
HI, @ICA3DaR5. Thanks for your work on this! We would like to bring this into the repo. To do this though, we cannot merge directing into rchain/rchain master branch. Please close this PR, create a new PR that requests merge into the dev branch, and assign me as a reviewer. I will help complete this process for you.

**Comment by ICA3DaR5** [comment]
@KellyatPyrofex Thanks a lot for your reply.

**Pull request closed without merging**